### PR TITLE
Add row limit warning

### PR DESCRIPTION
**Pull request by whscullin**
Fixes https://github.com/looker-open-source/malloy/issues/29

Screen Shot 2022-05-16 at 3 00 10 PM
1

### CODE REVIEW

**Comment by christopherswenson** [comment]
Looks great to me. I would check with @tschomer to see if he has an ideas about styling / placement, if you haven't already.

File: packages/malloy-vscode/src/extension/webviews/query_page/App.tsx [hunk 1]
```diff
@@ -98,6 +100,11 @@ export const App: React.FC = () => {
               setStatus(Status.Displaying);
               setTimeout(() => {
                 setHTML(rendered);
+                if (data.rowCount < result.totalRows) {
+                  setWarning(
+                    `Row limit hit. Viewing ${data.rowCount} of ${result.totalRows} results.`
```
Comment (christopherswenson) [inline packages/malloy-vscode/src/extension/webviews/query_page/App.tsx:105]: Might be nice to use `.toLocaleString()` for these to get the nice comma thousands separators.

Comment (christopherswenson) [inline packages/malloy-vscode/src/extension/webviews/query_page/App.tsx:105]: Might be nice to use `.toLocaleString()` for these to get the nice comma thousands separators.

**Comment by whscullin** [comment]
Screen Shot 2022-05-17 at 2 10 32 PM

**Pull request merged**